src/fides/api/ops/models/privacy_notice.py

```diff
@@ -0,0 +1,191 @@
+from __future__ import annotations
+
+from collections import defaultdict
+from enum import Enum
+from typing import Any, Dict, Iterable, List, Tuple
+
+from sqlalchemy import Boolean, Column
+from sqlalchemy import Enum as EnumColumn
+from sqlalchemy import Float, ForeignKey, String
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.orm import Session, relationship
+
+from fides.api.ops.common_exceptions import ValidationError
+from fides.lib.db.base_class import Base, FidesBase
+
+
+class PrivacyNoticeRegion(Enum):
+    """
+    Enum is not formalized in the DB because it is subject to frequent change
+    """
+
+    us_ca = "us_ca"
```

Comment:
probably want to double-check this before we merge?

Reply:
@adamsachs to cover ourselves and make things easier going forward - can we add a list item for each EU country? Reason being is that per Recital 10 of the GDPR:
> Member States should be allowed to maintain or introduce national provisions to further specify the application of the rules of this Regulation. In conjunction with the general and horizontal law on data protection implementing Directive 95/46/EC, Member States have several sector-specific laws in areas that need more specific provisions. This Regulation also provides a margin of manoeuvre for Member States to specify its rules, including for the processing of special categories of personal data (‘sensitive data’). To that extent, this Regulation does not exclude Member State law that sets out the circumstances for specific processing situations, including determining more precisely the conditions under which the processing of personal data is lawful.

In fact, we may have to account for Germany individually already. This could result in list items like `eu_de` and `eu_nl` for example.

Reply:
OK - that works for me @mfbrown . just to be clear, we will _not_ consider the EU member countries as overlapping with the broader `eu` region, though. if that's needed moving forward, we could look to add in that logic - but it feels out of scope for this initial body of work IMO.

i'm going to use [this](https://ec.europa.eu/eurostat/statistics-explained/index.php?title=Glossary:Country_codes) as a reference for country codes to support (e.g. `eu_nl`), hope that's ok? seems somewhat official :) if there's a better reference, please let me know!

Reply:
updated in https://github.com/ethyca/fides/pull/2928/commits/a8828f2da12b45f9b880aef91e96ffd2620a8158, new comment thread there so closing this out